In [1]:
from google.colab import drive
drive.mount('/gdrive')

#CSV파일로 변환해서 gdrive에 저장 하기 위해서!!

Go to this URL in a browser: https://accounts.google.com/o/oauth2/auth?client_id=947318989803-6bn6qk8qdgf4n4g3pfee6491hc0brc4i.apps.googleusercontent.com&redirect_uri=urn%3aietf%3awg%3aoauth%3a2.0%3aoob&response_type=code&scope=email%20https%3a%2f%2fwww.googleapis.com%2fauth%2fdocs.test%20https%3a%2f%2fwww.googleapis.com%2fauth%2fdrive%20https%3a%2f%2fwww.googleapis.com%2fauth%2fdrive.photos.readonly%20https%3a%2f%2fwww.googleapis.com%2fauth%2fpeopleapi.readonly

Enter your authorization code:
··········
Mounted at /gdrive


In [0]:
import pandas as pd

In [0]:
twit_cacao = pd.read_csv('/gdrive/My Drive/카카오 키워드 분석/카카오 트위터 크롤링.csv')

In [0]:
text = ""
for val in twit_cacao['내용']:
    text += val

In [0]:
text_list = text.replace("\n", " ").replace("\\n", " ").replace(":", " ").split()

In [0]:
keyword = set(text_list)

In [0]:
keyword = list(keyword)

In [38]:
keyword[:5]

['한세트', 'ㅅㅅㅇ', '👠밤을', '쉬러간다고', '먼']

In [0]:
word_count = []

for word in keyword:
    count = 0
    for word2 in text_list:
        if word == word2:
            count += 1
    word_count.append([word, count])

In [57]:
word_count #[키워드, 횟수]

[['한세트', 72],
 ['ㅅㅅㅇ', 144],
 ['👠밤을', 72],
 ['쉬러간다고', 147],
 ['먼', 72],
 ['//t.co/tVxCX1JCRURT', 72],
 ['정…RT', 72],
 ['중', 72],
 ['떠나보내기', 147],
 ['두분께', 71],
 ['참참참...?', 72],
 ['신속/안전거래◆◆전화번호', 72],
 ['또', 71],
 ['ㅠㅠ-카카오톡', 55],
 ['+1500', 143],
 ['귀걸이와', 72],
 ['반모', 288],
 ['마음을', 144],
 ['구독취소바,,.바보냐고', 72],
 ['이야기라도…RT', 142],
 ['구합니다(?)', 72],
 ['차트‼️', 71],
 ['그', 286],
 ['시우민편지봉투와', 71],
 ['???', 137],
 ['//t.co/rGALv1bhbY채용', 72],
 ['칭긔칭긔도', 213],
 ['튀다가', 9],
 ['여성', 72],
 ['카카오단톡방에서', 143],
 ['물론', 9],
 ['&lt;계좌번호&gt;', 72],
 ['머니게임', 72],
 ['▶▶▶전화번호', 72],
 ['(3set부터는', 143],
 ['롤듀오', 142],
 ['작별', 213],
 ['섞이지', 71],
 ['죽쟈', 9],
 ['안보셨나요?', 72],
 ['마사지', 72],
 ['싫어하는', 71],
 ['이게', 71],
 ['//t.co/ziezpTi9eP카카오그린의', 71],
 ['뭔데이건또', 66],
 ['어쩌고저쩌고', 72],
 ['.', 576],
 ['23', 308],
 ['네이버', 154],
 ['술', 143],
 ['여잔데', 9],
 ['에디션', 1117],
 ['》', 72],
 ['달이', 72],
 ['오…RT', 143],
 ['말하는건가?', 72],
 ['드림', 72],
 ['어린이날', 213],
 ['펜타블렛', 856],
 ['alltk', 244],
 ['실패', 9],
 ['\U0

In [0]:
while True:
    veri = 0
    for i in range(len(word_count)-1):
        if word_count[i][1] < word_count[i+1][1]:
            veri += 1
            word_count[i], word_count[i+1] = word_count[i+1], word_count[i]
    if veri == 0:
        break

- 카카오톡, 3741
- 카카오채널 : 379
- 카카오뱅크 : 287
- 멜론 : 226
- 오픈채팅방 : 219
- 네이버 : 154
- 카카오페이 : 143
- 카카오단톡방 : 143
- 정준영 : 143
- #카카오톡해킹#위치추적 : 142
- 게임 : 72
- 오픈카카오 : 72


In [62]:
3405 + 120 + 72 + 72 + 72

3741

In [60]:
word_count

[['https', 4643],
 ['카카오톡', 3405],
 ['5월', 2209],
 ['있는', 2045],
 ['이벤트', 2045],
 ['📢2020년', 1973],
 ['팔로우하고', 1973],
 ['@compury_jaewoo', 1973],
 ['//t.co/Yq1xUmxXPa', 1973],
 ['⁉️컴퓨리', 1973],
 ['수', 1973],
 ['리트윗하면', 1973],
 ['1명', 1973],
 ['참여완료!', 1973],
 ['할', 1973],
 ['추첨', 1973],
 ['상담', 1973],
 ['에디션', 1117],
 ['스위치', 1117],
 ['동숲', 1117],
 ['🎁닌텐도', 1117],
 ['🌟컴퓨리랑', 1117],
 ['10일', 1117],
 ['펜타블렛', 856],
 ['🎁휴이온', 856],
 ['가성비', 856],
 ['📱컴퓨리랑', 856],
 ['4일', 856],
 ['반박', 855],
 ['불소', 784],
 ['싫테', 784],
 ['임구', 784],
 ['톡디', 784],
 ['설참', 784],
 ['실매', 784],
 ['윰차', 784],
 ['카카오', 653],
 ['💕컴퓨리…RT', 650],
 ['🌟컴퓨리에서…RT', 646],
 ['아이디', 640],
 ['.', 576],
 ['반신', 568],
 ['기간', 546],
 ['소액결제현금', 539],
 ['싫어요', 502],
 ['테러', 502],
 ['구취', 496],
 ['전공', 496],
 ['-', 472],
 ['채널', 429],
 ['후', 361],
 ['불타는', 360],
 ['소통', 360],
 ['전국', 344],
 ['23', 308],
 ['프로필', 298],
 ['찾고있다.', 294],
 ['휴대폰소액결제', 292],
 ['핸드폰소액결제', 292],
 ['반모', 288],
 ['실시간', 288],
 ['그', 286],
 ['등', 286],
 

In [0]:
# 유의미한 단어 높은 순으로 뽑아보자
